test-resources-plugin/src/main/java/io/micronaut/gradle/testresources/MicronautTestResourcesPlugin.java

```diff
@@ -129,6 +123,23 @@ private void configurePlugin(Project project) {
         configureServiceReset((ProjectInternal) project, settingsDirectory, stopAtEndFile);
     }
 
+    private Path createStopFile(Project project) {
+        Path stopAtEndFile;
+        try {
+            File asFile = project.getLayout().getBuildDirectory().file("test-resources/" + UUID.randomUUID()).get().getAsFile();
+            if (asFile.getParentFile().isDirectory() || asFile.getParentFile().mkdirs()) {
+                asFile.deleteOnExit();
+                stopAtEndFile = asFile.toPath();
+                Files.deleteIfExists(stopAtEndFile);
+            } else {
+                throw new IOException("Could not create directory for test resources stop file");
```

Comment:
Would you please mention the problematic filename in the exception message?

Reply:
will do